apps/site/src/lib/api/model/verification-code.model.ts

```diff
@@ -5,17 +5,24 @@ import { rword } from "rword";
 import { formatErrors } from "../../../util/api";
 import { User } from "./user.model";
 
-export type VerificationCodeVariant = "login" | "email";
+export type VerificationCodeVariant = "login" | "email" | "linkWordpress";
 
 export type VerificationCodeProperties = {
   user: DBRef;
-  variant: VerificationCodeVariant;
   code: string;
   numberOfAttempts: number;
   used: boolean;
   createdAt: Date;
+  variant: VerificationCodeVariant;
+  wordpressInstanceUrl?: string;
 };
 
+// @note not using a discriminated union to match class properties, where that isn't possible
```

Comment:
This double negation is confusing me a little, what does this imply? :sweat_smile:

Reply:
It's trying to explain what I explained at https://github.com/blockprotocol/blockprotocol/pull/1231#discussion_r1153208059 

We can pair on this if you'd like?

Reply:
Nate and I chatted offline about this, see 5051b717c7b4e1f595b9868c4cc4340c6604e9c1